src/components/AvatarWithImagePicker.js

```diff
@@ -93,6 +98,23 @@ class AvatarWithImagePicker extends React.Component {
         this.animation.stop();
     }
 
+    /**
+     *  Toggle max upload limit modal's visibility
+     * @param {Boolean} isVisible
+    */
+    setUploadLimitModalVisibility(isVisible) {
+        this.setState({isMaxUploadSizeModalOpen: isVisible});
+    }
+
+    /**
+     * Check if the attachment size is less than allowed size.
+     * @param {Object} image
+     * @returns {Boolean}
+     */
+    isValidSize(image) {
+        return image && lodashGet(image, 'size', 0) < CONST.AVATAR_MAX_ATTACHMENT_SIZE;
```

Comment:
Why are we passing the size as a prop but then using the constant directly?

Reply:
@pecanoro `size` is for the size of the Avatar image - Large or default. Wasn't added in this PR. I just updated proptypes.

and for upload size limit I had added props but based on the feedback removed, so using CONST directly.